### ISSUE TITLE
conda installation issue

### ISSUE THREAD

**Issue by kellyrowland** [pyne/pyne#786]
Hello,

There seems to be an issue with the conda installer. Using

```
conda install -c pyne pyne
```

gives an error of

```
Fetching package metadata: ......
Solving package specifications: ...............
Error: Could not find some dependencies for pyne: lapack

Did you mean one of these?

    pyne

Did you mean one of these?

    clawpack

You can search for this package on anaconda.org with

    anaconda search -t conda lapack

 (and similarly for the other packages)
```

even though I have installed LAPACK using conda. However,

```
conda install -c cyclus -c pyne pyne=0.5.0
```

found [here](https://groups.google.com/forum/#!topic/pyne-users/xgELmysDTVM) seems to work:

```
Fetching packages ...
moab-4.8.1-0.t 100% |##########################################################| Time: 0:00:01   1.60 MB/s
pytaps-1.4-0.t 100% |##########################################################| Time: 0:00:00 389.61 kB/s
pyne-0.5.0-npp 100% |##########################################################| Time: 0:00:04   7.91 MB/s
Extracting packages ...
[      COMPLETE      ]|#############################################################################| 100%
Linking packages ...
[      COMPLETE      ]|#############################################################################| 100%
This is an install message from the pyne package.
```

**Comment by scopatz** [comment]
Thanks for reporting @kellyrowland 

> even though I have installed LAPACK using conda

What do you mean by this precisely?  Which other lapack package are you using?

**Comment by kellyrowland** [comment]
I installed lapack using conda with

```
conda install -c https://conda.binstar.org/cyclus lapack
```

**Comment by scopatz** [comment]
Ahh This is a conda proclivity.  Each time conda runs install or update it wants to know where to find package, in order to check that they are up-to-date. What you have installed currently matters only matters if it matches what could be installed.  When installing pyne, I think you need to always call it with the cyclus channel or put the cyclus and pyne channels in your condarc. We might want to put the lapack package in the pyne channel just to simplify things.

**Comment by kellyrowland** [comment]
resolved with #808.
